api/src/main/java/org/openmrs/api/db/hibernate/HibernateDiagnosisDAO.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.db.hibernate;
+
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.hibernate.Query;
+import org.hibernate.SessionFactory;
+import org.openmrs.Diagnosis;
+import org.openmrs.Encounter;
+import org.openmrs.Patient;
+import org.openmrs.api.db.DAOException;
+import org.openmrs.api.db.DiagnosisDAO;
+import org.springframework.transaction.annotation.Transactional;
+
+
+/**
+ * Hibernate implementation of the DiagnosisDAO
+ *
+ * @see DiagnosisDAO
+ * @see org.openmrs.api.DiagnosisService
+ *
+ */
+public class HibernateDiagnosisDAO implements DiagnosisDAO {
+	
+//	private static final Log log = LogFactory.getLog(HibernateDiagnosisDAO.class);
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode